quizinstruction.css

```diff
@@ -0,0 +1,28 @@
+.quiz-info-item{
+    display: flex;
+    justify-content: space-around;
+}
+.quiz-info{
+    margin-left: 10%;
+    margin-right: 1rem;
+}
+
+@media only screen and (max-width: 1080px){
+    .hero-section-container{
+        grid-column: 1 / 3;
+    }
+    .sidebar{
+        display: none;
+    }
+}
+@media only screen and (max-width: 750px){
```

Comment:
suggestion: you can use CSS variables for media queries.

Reply:
Yes i can use it but i wanna keep as this way for now